### ISSUE TITLE
On using absolute toc URLs

### ISSUE THREAD

**Issue by Asido** [futurepress/epub.js#451]
Suppose an epub has the following structure:
```
Content/ch1.xhtml
Navigation/toc.xhml
package.opf
```

`toc.xhtml` will contain a link `href="../Content/ch1.xhtml"` and `package.opf` will contain an item `href="Content/ch1.xhtml"` making `spineIndexByURL === { "Content/ch1.xhtml": 0 }` and therefore `spineIndexByURL[ book.toc[0].href ] === undefined`.

Wouldn't it make sense to convert all URLs to absolute? There are plenty of epubs having weird directory structures.

**Comment by fchasen** [comment]
Yes, that certainly makes sense. I've added URI.js to v0.3 so it should be easy to implement there and try to backport it.

**Comment by geek1011** [comment]
For me it is not even working to have a link to an xhtml file in the TOC. When I use book.displayChapter("chapter_001.xhtml") or book.goto("chapter_001.xhtml") it does not do anything. Btw, my code is in https://github.com/geek1011/BookReader/blob/master/App/js/reader.js#L190-L212
